### DISCUSSION TITLE
How are throat lengths calculated?

### DISCUSSION THREAD

**Discussion by L278** [PMEAL/OpenPNM#1937] [Q&A]
Hi, when finding the throat lengths using one of the models like cones_and_cylinders or cubes_and_cuboids etc, is the throat length calculated from the surface of the pores with their corresponding shape?

**Comment by ma-sadeghi** [comment]
Hi, here are some descriptive sketches:

# `spheres_and_cylinders`
<img src="https://user-images.githubusercontent.com/14086031/124921787-c665a180-dfc6-11eb-9dc3-3872d984f58f.png" width=50%>

# `cubes_and_cuboids`
<img src="https://user-images.githubusercontent.com/14086031/124921774-bfd72a00-dfc6-11eb-93e4-dfee87de8d1d.png" width=50%>

# `cones_and_cylinders`
<img src="https://user-images.githubusercontent.com/14086031/124921737-b64dc200-dfc6-11eb-9427-ed579d983bad.png" width=50%>
